src/style.css

```diff
@@ -0,0 +1,119 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+:root {
+  --outline: 0;
+  --border: 1px solid #e1e7ed;
+  --border-radius: 4px;
+  --main-text-color: #3d4e61;
+  --font-family: "Avenir", sans-serif;
+}
+
+html {
```

Comment:
Consider setting a default font size for better accessibility and user experience. 📐